client/transactions/list/deposit.js

```diff
@@ -26,13 +25,17 @@ const Deposit = ( { depositId, dateAvailable } ) => {
 			true // TODO Change call to gmdateI18n and remove this deprecated param once WP 5.4 support ends.
 		);
 
-	return depositId ? (
+	const estimated =
+		depositId && depositId.includes( 'wcpay_estimated_' )
+			? 'Estimated'
```

Comment:
`'Estimated'` should be a translated string here. Use `__` similar to the removed `'Pending'` indicator.